gui/start/start_gui.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+source ~/owr-qtgui/devel/setup.bash
```

Comment:
Rather than doing this which can cause problems do this:

Basically those `setup.bash` files don't stack. The one you've sourced most recently will be the one it uses. 
To get them to stack what you have to do is source the first workspace before building the second workspace (which will add the first one to its path) then source the second workspace